.mdox.validate.yaml

```diff
@@ -34,6 +34,13 @@ validators:
   # Ignore anchor links pointing to the API documentation which are HTML <a> tags and not supported by mdox.
   - regex: 'api\.md#monitoring\.coreos\.com/v1\.(BasicAuth|PrometheusSpec|StorageSpec)$'
     type: ignore
+  # Ignore dead links from Ambassador (soon to be removed).
+  - regex: 'getambassador'
+    type: ignore
+  # Ignore all github.com URLs because of rate limiting.
+  # TODO: find an alternative way to check these URLs avoiding the rate limiting.
+  - regex: 'github.com'
+    type: ignore
```

Comment:
```suggestion
  # Ignore dead links from Ambassador (soon to be removed).
  - regex: 'getambassador'
    type: ignore
  # Ignore all github.com URLs because of rate limiting.
  # TODO: find an alternative way to check these URLs avoiding the rate limiting.
  - regex: 'github.com'
    type: ignore
  - regex: 'istio.io'
    type: ignore
```

Reply:
Add `istio.io` URL

Reply:
hmm istio.io doesn't seem to cause issues?

Reply:
```
2 errors:
		Documentation/proposals/202305-scrapeclasses.md:194: "https://istio.io/latest/docs/reference/config/security/authorization-policy/" not accessible even after retry; status code 0: Get "https://istio.io/latest/docs/reference/config/security/authorization-policy/": http2: timeout awaiting response headers
		Documentation/proposals/202305-scrapeclasses.md:189: "https://istio.io/latest/docs/concepts/security/#permissive-mode" not accessible even after retry; status code 0: Get "https://istio.io/latest/docs/concepts/security/#permissive-mode": http2: timeout awaiting response headers
```
Hmmm.... Adjust the timeout period? 🤔

Reply:
istio causes issues sometimes but should be ok for now as re-run helps to pass